pydis_site/apps/api/migrations/0025_allow_custom_inserted_at_infraction_field.py

```diff
@@ -14,6 +14,6 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='infraction',
             name='inserted_at',
-            field=models.DateTimeField(default=datetime.datetime.utcnow, help_text='The date and time of the creation of this infraction.'),
+            field=models.DateTimeField(default=timezone.now(), help_text='The date and time of the creation of this infraction.'),
```

Comment:
You should provide the callable so Django can use it to generate a default when it needs it. If you call it in the definition, it will be a single, static default value.

```suggestion
            field=models.DateTimeField(default=timezone.now, help_text='The date and time of the creation of this infraction.'),
```

Reply:
good catch.